cookiecutter/{{cookiecutter.github_repo_name}}/CONTRIBUTING.md

```diff
@@ -0,0 +1,101 @@
+# Contributing to {{cookiecutter.title}}
+
+## Build
+
+You can use `build` to simply build the source and binary distribution:
+
+```sh
+python -m pip install build
+python -m build
+```
+
+## Local development
+
+You can use editable installs to develop the project locally (it will install
+all the dependencies too):
+
+```sh
+python -m pip install -e .
+```
+
+Or you can install all development dependencies (`mypy`, `pylint`, `pytest`,
+etc.) in one go too:
+```sh
+python -m pip install -e .[dev]
+```
+
+If you don't want to install all the dependencies, you can also use `nox` to
+run the tests and other checks creating its own virtual environments:
+
+```sh
+python -m pip install .[dev-noxfile]
+nox
+```
+
+You can also use `nox -R` to reuse the current testing environment to speed up
+test at the expense of a higher chance to end up with a dirty test environment.
+
+### Running tests / checks individually
+
+For a better development test cycle you can install the runtime and test
+dependencies and run `pytest` manually.
+
+```sh
+python -m pip install .[dev-pytest]  # included in .[dev] too
+
+# And for example
+pytest tests/test_*.py
+```
+
+Or you can use `nox`:
+
+```sh
+nox -R -s pytest -- test/test_*.py
+```
+
+The same appliest to `pylint` or `mypy` for example:
+
+```sh
+nox -R -s pylint -- test/test_*.py
+nox -R -s mypy -- test/test_*.py
+```
+
+## Releasing
+
+These are the steps to create a new release:
+
+1. Get the latest head you want to create a release from.
+
+2. Update the `RELEASE_NOTES.md` file if it is not complete, up to date, and
+   clean from template comments (`<!-- ... ->`) and empty sections. Submit
```

Comment:
```suggestion
replace template comments (`<!-- ... ->`) and empty sections with `None`. Submit
```

Reply:
I think we are mixing up styles here. Originally the `RELEASE_NOTES.md` template was thought to completely remove the empty sections, to remove the extra noise of just having and empty section (or as you suggest `None`).

There is a proposal to improve this (https://github.com/frequenz-floss/frequenz-repo-config-python/discussions/52), but I would leave it for a separate PR. The idea of this PR is just to bring `repo-config` on par with the current status quo.

Reply:
> There is a proposal to improve this (https://github.com/frequenz-floss/frequenz-repo-config-python/discussions/52), but I would leave it for a separate PR. The idea of this PR is just to bring repo-config on par with the current status quo.

Fair enough.
I'd still suggest the following fix: "clean from" => "remove" ("clean from" does not make much sense here)

Reply:
Done.